src/iarc7_motion/iarc_tasks/track_roomba_task.py

```diff
@@ -83,6 +83,7 @@ def __init__(self, task_request):
 
     def get_desired_command(self):
         with self._lock:
+
```

Comment:
Possible to remove the cosmetic line addition? Otherwise I think you might run into merge issues when you rebase.

Reply:
Removed that line

Reply:
Fixed this.

Reply:
For future reference, removing changes like this will not make the rebase easier, if anything you'll have to fix the merge twice as many times (unless, to remove the change, you amend the commit instead of adding a new commit).

Reply:
Yeah he’d have to squash his commits.